pkg/converter/convert.go

```diff
@@ -199,7 +200,8 @@ func Convert(o Options, pathOptions *clientcmd.PathOptions) error {
 			Args: []string{
 				getTokenCommand,
 			},
-			APIVersion: execAPIVersion,
+			APIVersion:  execAPIVersion,
+			InstallHint: execInstallHint,
```

Comment:
let's use the install hint from input, when provided, before falling back to the hint here

Reply:
Thanks for the suggestion @weinong. I think my latest push contains the changes you wanted. PTAL.